### PR TITLE
Fix cayuse_pi_name ValueTooLong error

### PR DESCRIPTION
**Pull request by 6reg**
https://www.pivotaltracker.com/story/show/182884178

### CODE REVIEW

File: app/models/epds/cayuse_project.rb [hunk 1]
```diff
@@ -23,15 +23,21 @@ class Epds::CayuseProject < Epds::Connection
   self.primary_key = "PROJECT_ID"
 
   has_many :cayuse_awards, foreign_key: 'PROJECT_ID'
-  has_many :cayuse_research_teams, -> { where(ROLE: 'Lead Principal Investigator') }, through: :cayuse_awards 
+  has_many :cayuse_research_teams, through: :cayuse_awards #-> { where(ROLE: 'Lead Principal Investigator') },
```
Comment (amcates) [inline app/models/epds/cayuse_project.rb:26]: I believe this `where` statement is still needed.  We don't want all of the users.  Just the Lead Principal Investigators.  If that list is more than 1, show the `Multiple Principal Investigators` message.  If you remove the where you will be getting all types of people.

**Comment by 6reg** [comment]
@amcates can I merge this?

**Comment by amcates** [comment]
@6reg No, only someone else (like myself) will merge and deploy pull request by another author

**Comment by 6reg** [comment]
I don't see this on production

**Pull request merged**